test/integration_test/specs/migration/migration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: Migration
+metadata:
+  name: mysql-migration
+spec:
+  # This should be the name of the cluster pair created above
+  clusterPair: remoteclusterpair
+  # If set to false this will migrate only the Portworx volumes. No PVCs, apps, etc will be migrated
+  includeResources: true
+  # If set to false, the deployments and stateful set replicas will be set to 0 on the destination. There will be an annotation with "stork.openstorage.org/migrationReplicas" to store the replica count from the source
+  startApplications: true
+  # List of namespaces to migrate
+  namespaces:
+  - mysql-1-pvc-singlemysql 
```

Comment:
How does this work? torpedo generates a random name for the namespace

Reply:
torpedo does not generates random name, we supply it while scheduling using **generateInstanceID** . I'm simply using **singlemysql** as instanceID ,which torpedo then forms namespace as  **mysql-1-pvc-singlemysql**  (i.e specKey-instanceID)